### DISCUSSION TITLE
Is there a number input in native-base like this?

### DISCUSSION THREAD

**Discussion by radiorz** [GeekyAnts/NativeBase#5118] [Q&A]
![image](https://user-images.githubusercontent.com/24271959/176876301-32027ad3-b5c3-49f3-acba-084da711e176.png)

**Comment by radiorz** [comment]
I find that there are numberinput in nativebase like this: 
```
          <NumberInput onChange={val => item.onChange(val)} defaultValue={item.value}>
            <NumberInputField w={100} />
            <NumberInputStepper>
              <NumberIncrementStepper />
              <NumberDecrementStepper />
            </NumberInputStepper>
          </NumberInput>
```

![image](https://user-images.githubusercontent.com/24271959/176876534-d91026b3-1652-4a4d-bb7f-689e62f877f3.png)

**Reply by radiorz** [reply]
How to change its ui?

**Reply by sanna231** [reply]
We had it in the alpha versions of v3 but due to some accessibility issues, we aren't supporting it right now. It's in our plans and we will release it later.
